src/components/dashboard/components/concepts/ConceptModal.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AddConceptModal = props => (
```

Comment:
The modal shows concept details, could you change the name to reflect that instead of AddConceptModal

Reply:
It is actually where the user clicks to add that particular concept

Reply:
Alright then that makes sense